content/kotlin/concepts/queue/queue.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: 'Queue'
+Description: 'A Queue is a collection interface that offers additional operations for accessing or manipulating items at the head of the queue.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Collections'
+  - 'Queues'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+A `queue` is a data structure that stores elements in a linear order, and follows the First In, First Out (FIFO) principle. This means that the first element added to the queue will be the first one to be removed.
+
+### Syntax
+
+The syntax for creating a queue in Kotlin using the LinkedList class is as follows :
+
+```pseudo
+val queue = LinkedList<Type>()
+```
+
+Where Type is the type of elements that the queue will hold, for example, if you want to store integers, you can use Int as the type :
+
+```pseudo
+val queue = LinkedList<Int>()
+```
+
+You can also use the Queue interface which is implemented by LinkedList :
+
+```pseudo
+val queue = LinkedList<Type>().toQueue()
+```
+
+### Example
+
+```kotlin
+// Create a new queue
+val queue = LinkedList<Int>()
+
+// Add elements to the queue
+queue.add(1)
+queue.add(2)
+queue.add(3)
+
+// Remove elements from the queue
+val firstElement = queue.remove() // Returns 1
+val secondElement = queue.remove() // Returns 2
+```
+
+Alternatively, you can use Queue interface which is implemented by LinkedList :
```

Comment:
Remove this and the example. It's redundant with the syntax section.

Reply:
alright